GUI.py

```diff
@@ -0,0 +1,93 @@
+import tkinter as tk
+from tkinter import ttk
+from recipe import Recipe
+from recipemanager import RecipeManager
+import tkinter.filedialog
+#from recipemanager import Recipe
+
+"""
+# class Main_Application(tk.Frame):
+#     def __init__(self,parent,*args,**kwargs):
+#         tk.Frame.__init__(self,parent,*args,**kwargs)
+#         self.parent = parent
+
+# if __name__ == "__main__":
+#     root = tk.Tk()
+#     Main_Application(root).pack(side="top", fill="both", expand=True)
+#     root.mainloop()
+"""
+
+   
+root = tk.Tk()
+#lbl_recipe_listbox = tk.Label(text="Here are your recipes:")
+recipe_manager = RecipeManager()
+recipe_manager.recipes = ['pie', 'tart', 'lasagne']
+
+def display_recipes():
+    main_listbox()
+
+#Create a Listbox
+def main_listbox():
+    list_items=tk.Variable(value= recipe_manager.recipes)
+    listbox = tk.Listbox(
+        root,
+        height = 10,
+        listvariable = list_items,
+        selectmode=tk.SINGLE
+    )
+    #Formatting listbox so it fills screen
+    listbox.pack(expand=True,fill=tk.BOTH, side=tk.LEFT)
+
+    scrollbar = ttk.Scrollbar(
+        root,
+        orient=tk.VERTICAL,
+        command=listbox.yview
+    )
+    listbox["yscrollcommand"] = scrollbar.set
+    scrollbar.pack(side=tk.LEFT,expand=True, fill=tk.Y)
+
+
+    def items_selected(event):
+        #gets selected item
+        selected_indices  = listbox.curselection()
+        selected_recipes=",".join([listbox.get(i) for i in selected_indices])
+        msg=f"You selected: {selected_recipes}"
+        chosen_item = tk.Label(text=msg)
+        chosen_item.pack()
+
+    listbox.bind('<<ListboxSelect>>', items_selected)
+
+
+#The below is direcetory selector on uml but need to look into filedialog again
+
+
+display_recipes()
+#tkinter.filedialog.askdirectory() #This pops up the file explorer
+ent_recipe = tk.Entry(fg="blue",bg = "pink", width=50)
+ent_recipe.pack()
+
+requested_recipe = ent_recipe.get()
```

Comment:
I am not sure about this line, probably I might underestand it wrong, but that can not catch user input. Probabaly what I will try to do is:

def submit_recipe():
    requested_recipe = ent_recipe.get()

Reply:
Yes I couldn't figure out how to catch user input so that sounds like a good idea

Reply:
I get a circular reference when I try to launch the GUI